tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py

```diff
@@ -255,14 +255,18 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
 
                 p_mask = self.fill_p_mask(span, len(truncated_query), sequence_added_tokens, cls_index)
                 idx = example_index
-                identifier = ['input_ids_{}'.format(idx), 'input_mask_{}'.format(idx), 'segment_ids_{}'.format(idx)]
+                identifier = ['input_ids_{}'.format(idx),
+                              'input_mask_{}'.format(idx),
+                              'segment_ids_{}'.format(idx),
+                              'position_ids_{}'.format(idx)]
                 annotation = QuestionAnsweringAnnotation(
                     identifier,
                     example.qas_id,
                     np.array(unique_id),
                     np.array(span["input_ids"]),
                     np.array(span["attention_mask"]),
                     np.array(span["token_type_ids"]),
+                    np.arange(len(span["token_type_ids"])),#position_ids
```

Comment:
Instead of a comment, I would suggest to use a named argument.

Reply:
ok